source/simulator/solver_schemes.cc

```diff
@@ -678,6 +678,12 @@ namespace aspect
 
     if (parameters.run_postprocessors_on_nonlinear_iterations)
       postprocess ();
+
+    // Setup a nonlinear solver control that only allows a single iteration
+    SolverControl nonlinear_solver_control(1,1.0);
+    // Announce that we did a single iteration, and assume we have converged
+    nonlinear_solver_control.check(0,0.0);
```

Comment:
I think you want
```suggestion
    nonlinear_solver_control.check(1, 0.0);
```
to indicate that *one* iteration was actually done.